.editorconfig

```diff
@@ -147,6 +147,10 @@ dotnet_naming_style.pascal_case_style.capitalization = pascal_case
 # error RS2008: Enable analyzer release tracking for the analyzer project containing rule '{0}'
 dotnet_diagnostic.RS2008.severity = none
 
+# RS1034: Prefer 'IsKind' for checking syntax kinds
+# Hold on https://github.com/dotnet/roslyn/pull/51823#pullrequestreview-612747491
+dotnet_diagnostic.RS1034.severity = none
```

Comment:
❗ Use .globalconfig instead of .editorconfig for disabling rules (the latter requires evaluating the severity condition for every syntax tree in the solution, instead of just once). Roslyn's global configuration files are here:
https://github.com/dotnet/roslyn/tree/main/eng/config/globalconfigs